src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -298,6 +304,10 @@ private ZipFileProcessingResult processZipFileContent(
             result.setEnvelope(envelopeProcessor.saveEnvelope(dbEnvelope));
 
             return result;
+        } catch (InvalidJourneyClassificationException ex) {
```

Comment:
`InvalidJourneyClassificationException` no longer needs to be `InvalidEnvelopeException` extension ;)